apps/src/lib/ui/simpleSignUp/workshop/WorkshopWelcomeBanner.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import Typography from '@cdo/apps/componentLibrary/typography/Typography';
+import i18n from '@cdo/locale';
+
+import codeLogo from '../assets/codeLogo.svg';
+
+import styles from '../link-account.module.scss';
+
+const WorkshopWelcomeBanner = () => {
+  return (
+    <div className={styles.welcomeContainer}>
+      <div className={styles.welcomeIconContainer}>
+        <img src={codeLogo} alt={i18n.codeLogo()} />
+      </div>
+
+      <div className={styles.titleContainer}>
+        <Typography semanticTag={'h1'} visualAppearance={'heading-xxl'}>
+          {i18n.accountWelcomeBannerHeaderLabel()}
+        </Typography>
+        <Typography
+          className={styles.titleDesc}
+          semanticTag={'p'}
+          visualAppearance={'body-two'}
+        >
+          {i18n.accountWelcomeBannerContentWorkshopEnroll()}
+        </Typography>
```

Comment:
Nit: this can be also be simplified with `<BodyTwoText>` similar to the H1 above: 
```suggestion
        <BodyTwoText className={styles.titleDesc}>
          {i18n.accountWelcomeBannerContentWorkshopEnroll()}
        </BodyTwoText>
```